src/Andoromeda.Kyubey.Portal/Services/FileHelper.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Andoromeda.Kyubey.Portal.Services
+{
+    public static class FileHelper
+    {
+        public static void RemoveDangerousChar(this string fileName)
+        {
+            fileName = fileName.Replace(".", "").Replace("\\", "").Replace("/", "");
```

Comment:
What's this?

Reply:
this is filter illegal string like this "../../../windows/"